modules/base/markdown.go

```diff
@@ -250,6 +250,7 @@ func PostProcessMarkdown(rawHtml []byte, urlPrefix string) []byte {
 func RenderMarkdown(rawBytes []byte, urlPrefix string) []byte {
 	result := RenderRawMarkdown(rawBytes, urlPrefix)
 	result = PostProcessMarkdown(result, urlPrefix)
+	Sanitizer.AllowAttrs("class").Matching(regexp.MustCompile(`[\p{L}\p{N}\s\-_',:\[\]!\./\\\(\)&]*`)).OnElements("code")
```

Comment:
Yes. It only needs to be done once. So maybe it is better to do it in other place. I tried in `tools.go` just after the line that initializes it but I got some error about that line of code not being allowed there outside a function and I inmediately thought my best option was to put it just in markdown.go. I'm too new to go to know a better place to put that line of code but probably there is a better place indeed.